includes/class-newspack-popups-api.php

```diff
@@ -362,6 +371,21 @@ public function get_mailing_list_status( $request ) {
 		return 'subscribed' === $mailing_list_status;
 	}
 
+	/**
+	 * Get email address.
+	 *
+	 * @param WP_REST_Request $request amp-access request.
+	 * @return string Email address.
+	 */
+	public function get_email_address( $request ) {
+		$email_address = isset( $request['email'] ) ? esc_attr( $request['email'] ) : false;
+		if ( ! $email_address ) {
+			$body          = json_decode( $request->get_body(), true );
+			$email_address = isset( $body['email'] ) ? $body['email'] : false;
```

Comment:
Shouldn't `$body['email']` be escaped like `$request['email']`?

Reply:
Fixed in 7d6870866987d3356ccb19bfadd47bec4cd954a0